app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -187,6 +187,12 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             super.onCreate(savedInstanceState)
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && isInPictureInPictureMode) {
+            // Exit PiP mode
+            moveTaskToBack(false)
+            startActivity(Intent(this, this::class.java).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK))
```

Comment:
issue (non-blocking): I'm not sure how results in us exiting PiP mode.

suggestion: can you explain in the comment how this lets us exit PiP mode?

Reply:
https://stackoverflow.com/a/43288507/12204281
The code moves the activity to the back of the activity stack, then restores it to the front, bringing it out of PiP mode.

Reply:
I'm not sure I understand why moving the activity to the back of the activity stack and restoring it to the front would bring it out of PiP – it seems like it might be a hack. However, if it seems like this is the only way to do it, I'm fine with that – just link to the Stackoverflow post in a comment so others have context on where this solution came from.

In any case, I don't need to approve or reject the correctness changes: I'll defer to @Mugurell for those. @Mugurell Please finish reviewing – it looks good from my perspective on performance! Sorry if I was unclear about my intent.